src/components/filterTab/FilterTab.js

```diff
@@ -1,28 +1,38 @@
 import TabItem from "./TabItem";
-import { useLocation } from "react-router-dom";
-import { tabs } from "./constants";
+import { allTab } from "./constants";
 import SortMoviesSelect from "./SortMoviesSelect";
+import { movieGenre } from "../../constants/genre";
+import { useSearchParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { queryParamsValues, queryParamsKeys } from "../../constants/searchParams";
 
 const FilterTab = () => {
-  let location = useLocation();
+  const [searchParams] = useSearchParams();
+  const [currentParams, setCurrentParams] = useState(null);
+
+  useEffect(() => {
+    setCurrentParams(searchParams.get(queryParamsKeys.SEARCH));
+  }, [searchParams]);
+  const tabs = [...allTab, ...movieGenre().slice(0, 5)];
 
   return (
     <div className="text-center text-xs	text-white font-light border-b border-gray-200  flex justify-between ">
       <ul className="flex flex-wrap -mb-px">
         <li className="mr-2">
           {tabs.map((item, i) => (
             <TabItem
-              key={item.name}
-              path={item.path}
-              active={location.hash === item.path.hash}
+              key={i}
```

Comment:
good practice is not to use index but use some uniq item prop (id, hash, etc)

Reply:
@ambroseus  in this case, this array is not dynamic ( I understand your concern )